From f1f67d0db21226768829dff4097a7d95affff3a6 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sat, 25 Aug 2018 11:40:45 +0300
Subject: [PATCH] armcp: enhance TARGET_BUILD to support Gaudi

Note that HL_DEV is use so no more capiltalized string
---

diff --git a/Makefile b/Makefile
index 239db9d..fa9beba 100644
--- a/Makefile
+++ b/Makefile
@@ -122,7 +122,7 @@
 	@rsync -au --exclude .git $(BR2_EXTERNAL)/../3rd-parties/spdlog/ ${OUTPUT_DIR}/build/spdlog-custom
 	@rsync -au --exclude .git $(BR2_EXTERNAL)/../armcp/ ${OUTPUT_DIR}/build/armcp-custom
 	@mkdir -p ${OUTPUT_DIR}/build/armcp-custom/build
-	@cd ${OUTPUT_DIR}/build/armcp-custom/build && cmake -DTARGET_BUILD:STRING=Fpga ../
+	@cd ${OUTPUT_DIR}/build/armcp-custom/build && cmake -DTARGET_BUILD:STRING=fpga ../
 	@cd ${OUTPUT_DIR}/build/armcp-custom/build && make -j 4
 dali-rpi-gpio-mdio:
 	@cd && build_and_insmod_gpio_mdio
diff --git a/package/armcp/armcp.mk b/package/armcp/armcp.mk
index 8ebc954..16e1cf1 100644
--- a/package/armcp/armcp.mk
+++ b/package/armcp/armcp.mk
@@ -5,6 +5,7 @@
 ARMCP_DEPENDENCIES = specs spdlog
 
 ARMCP_CONF_OPTS = -DTESTS_ENABLED=OFF
+ARMCP_CONF_OPTS = -DTARGET_BUILD:STRING=$(HL_DEV)
 
 define ARMCP_INSTALL_TARGET_CMDS
         $(INSTALL) -D -m 0755 $(@D)/bin/armcpd $(TARGET_DIR)/usr/bin/armcpd
